pulumi/infra/postgres.py

```diff
@@ -140,9 +140,16 @@ def __init__(
             # Makes teardown easier. This can be revisited.
             skip_final_snapshot=True,
             # Eventually this would be managed by Vault.
-            # In the mean time, security is basically only enforced by VPC
+            # In the meantime, security is basically only enforced by VPC
             username=username,
             password=password.result,
             port=postgres_port,
-            opts=child_opts,
+            # Because we're not specifying a kms key, this will use the default aws/rds key
+            storage_encrypted=True,
+            # Enable performance insights for 7 days, which is free
+            performance_insights_enabled=True,
+            performance_insights_retention_period=7,
+            # We need delete before replace in order to encrypt instances. Once prod is up and running we may want to
+            # remove this option to avoid accidental downtime.
```

Comment:
Might be good to file an issue for this, just so we don't forget! _That_ would be a fun incident to deal with :joy: